easybuild/easyconfigs/d/DP3/DP3-6.0-foss-2023b.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'CMakeMake'
+
+name = 'DP3'
+version = '6.0'
+
+homepage = 'https://dp3.readthedocs.io/'
+description = "DP3: streaming processing pipeline for radio interferometric data."
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+sources = [
+    {
+        'filename': '%(name)s-v%(version)s.tar.gz',
+        'git_config': {
+            'url': 'https://git.astron.nl/RD',
+            'repo_name': '%(name)s',
+            'tag': 'v%(version)s',
+            'clone_into': '%(name)s',
+            'recursive': True
+        }
+    },
+]
+checksums = [None]
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+]
+dependencies = [
+    ('casacore', '3.5.0'),
+    ('Boost', '1.83.0'),
+    ('CFITSIO', '4.3.1'),
+    ('WCSLIB', '7.11'),
+    ('GSL', '2.7'),
+    ('HDF5', '1.14.3'),
+    ('Python', '3.11.5'),
+    ('EveryBeam', '0.5.2'),
+    ('Armadillo', '12.8.0'),
+    ('AOFlagger', '3.4.0'),
+]
+
+# Without this, the wrong (system) python is picked up by cmake
+# in cases where the system python is newer than the one used in these EasyConfigs
+# See PR # 19119
+configopts = "-DPython3_EXECUTABLE=$EBROOTPYTHON/bin/python "
```

Comment:
This should now be fixed in the CMakeMake EasyBlock by https://github.com/easybuilders/easybuild-easyblocks/pull/3088 . Please remove this configopts addition and check it still works (we can check when building for EESSI, where you found the issue previously....)